src/core/containers.py

```diff
@@ -23,14 +20,39 @@ def as_dict(self):
 
 
 class TaskTarget(Container):
-    __slots__ = [
-        "flow",
-        "task",
-    ]
+    __slots__ = ["flow", "task"]
 
     def __init__(self, flow_name, task_name):
         self.flow = flow_name
         self.task = task_name
 
     def as_dict(self):
         return {"flow": self.flow, "task": self.task}
+
+
+class Populate(Container):
+    __slots__ = ["path", "validators", "value"]
+
+    def _validate_args(self, path, value):
+        if value and path:
```

Comment:
value could be 0 so ```is not None``` is what you want to check

Reply:
done

Reply:
done